410musicdsl/ast/TEMPO.py

```diff
@@ -1,13 +1,17 @@
 from libs.node import Node
 from ast.INTEGER import INTEGER
+from ast.APP_VISITOR import Visitor
 
 
 class TEMPO(Node):
-    # FIELDS:
-    # tempo
-
-    def parse(self):
-        self.tokenizer.getAndCheckNext("Tempo:")
-        self.tempo = INTEGER()
```

Comment:
I thought we might be able to save some repetition by introducing these value classes, but if they are causing evaluation problems, we can remove!